site/content/blog/concourse-ytt.md

```diff
@@ -0,0 +1,342 @@
+---
+title: "YTT Tutorial: Converting Concourse pipeline to YTT"
+slug: concourse-ytt-101
+date: 2022-06-16
+author: Neil Hickey
+excerpt: "Tutorial: Concourse to YTT 101"
+tags: ['ytt', 'concourse']
+---
+
+Concourse is an automation system written in Go. It is most commonly used for CI/CD, and is built to scale to any kind of automation pipeline, from simple to complex. Each pipeline in Concourse is a declarative YAML file which represents input, tasks and output. Concourse pipelines often grow more complex as time goes on, and you may quickly find yourself overwhelmed trying to manage these large, complex pipelines. 
+
+Ytt is Carvel's tool of choice when it comes to managing YAML, and in this tutorial we will work through a few ways I like to think about making YAML more maintainable and readable using a simple Concourse pipeline.
+
+## You will need these to follow along:
+- ytt (https://carvel.dev/ytt/docs/latest/install/)
+- concourse (https://concourse-ci.org/install.html)
+- docker-compose / docker (optional)
+
+## Getting set up
+
+1. Install a local Concourse server (optional)
+
+```
+wget https://concourse-ci.org/docker-compose.yml && docker-compose up -d
+
+fly --target tutorial login --concourse-url http://localhost:8080 -u test -p test
+```
```

Comment:
```suggestion
    ```
    wget https://concourse-ci.org/docker-compose.yml && docker-compose up -d
    
    fly --target tutorial login --concourse-url http://localhost:8080 -u test -p test
    ```
```

suggestion: indent to preserve the ordered numbering (and it can be argued to improve the layout).